containers/general/tool.go

```diff
@@ -14,23 +14,24 @@ import (
 	"time"
 
 	"github.com/refractionPOINT/go-uspclient"
-	"github.com/refractionPOINT/usp-adapters/1password"
-	"github.com/refractionPOINT/usp-adapters/azure_event_hub"
-	"github.com/refractionPOINT/usp-adapters/duo"
-	"github.com/refractionPOINT/usp-adapters/evtx"
-	"github.com/refractionPOINT/usp-adapters/file"
-	"github.com/refractionPOINT/usp-adapters/gcs"
-	"github.com/refractionPOINT/usp-adapters/o365"
-	"github.com/refractionPOINT/usp-adapters/pubsub"
-	"github.com/refractionPOINT/usp-adapters/s3"
-	"github.com/refractionPOINT/usp-adapters/simulator"
-	"github.com/refractionPOINT/usp-adapters/slack"
-	"github.com/refractionPOINT/usp-adapters/sqs"
-	"github.com/refractionPOINT/usp-adapters/sqs-files"
-	"github.com/refractionPOINT/usp-adapters/stdin"
-	"github.com/refractionPOINT/usp-adapters/syslog"
+	usp_1password "github.com/refractionPOINT/usp-adapters/1password"
+	usp_azure_event_hub "github.com/refractionPOINT/usp-adapters/azure_event_hub"
+	usp_duo "github.com/refractionPOINT/usp-adapters/duo"
+	usp_evtx "github.com/refractionPOINT/usp-adapters/evtx"
+	usp_file "github.com/refractionPOINT/usp-adapters/file"
+	usp_gcs "github.com/refractionPOINT/usp-adapters/gcs"
+	usp_itglue "github.com/refractionPOINT/usp-adapters/itglue"
+	usp_o365 "github.com/refractionPOINT/usp-adapters/o365"
+	usp_pubsub "github.com/refractionPOINT/usp-adapters/pubsub"
+	usp_s3 "github.com/refractionPOINT/usp-adapters/s3"
+	usp_simulator "github.com/refractionPOINT/usp-adapters/simulator"
+	usp_slack "github.com/refractionPOINT/usp-adapters/slack"
+	usp_sqs "github.com/refractionPOINT/usp-adapters/sqs"
+	usp_sqs_files "github.com/refractionPOINT/usp-adapters/sqs-files"
+	usp_stdin "github.com/refractionPOINT/usp-adapters/stdin"
+	usp_syslog "github.com/refractionPOINT/usp-adapters/syslog"
```

Comment:
No need to name those imports, it will use the package name automatically.
```suggestion
	"github.com/refractionPOINT/usp-adapters/1password"
	"github.com/refractionPOINT/usp-adapters/azure_event_hub"
	"github.com/refractionPOINT/usp-adapters/duo"
	"github.com/refractionPOINT/usp-adapters/evtx"
	"github.com/refractionPOINT/usp-adapters/file"
	"github.com/refractionPOINT/usp-adapters/gcs"
	"github.com/refractionPOINT/usp-adapters/itglue"
	"github.com/refractionPOINT/usp-adapters/o365"
	"github.com/refractionPOINT/usp-adapters/pubsub"
	"github.com/refractionPOINT/usp-adapters/s3"
	"github.com/refractionPOINT/usp-adapters/simulator"
	"github.com/refractionPOINT/usp-adapters/slack"
	"github.com/refractionPOINT/usp-adapters/sqs"
	"github.com/refractionPOINT/usp-adapters/sqs-files"
	"github.com/refractionPOINT/usp-adapters/stdin"
	"github.com/refractionPOINT/usp-adapters/syslog"
```